Sources/Paywalls/Components/PaywallV2CacheWarming.swift

```diff
@@ -87,9 +87,12 @@ extension PaywallComponentsData.PaywallComponentsConfig {
                 urls += carousel.pages.flatMap({ stack in
                     self.collectAllImageURLs(in: stack)
                 })
-            case .video:
-                // WIP: - prewarm cache
-                break
+            case .video(let video):
+                Task {
+                    // WIP: Need to inject the file repository so we can potentially share the work in the view layer
```

Comment:
This is an optimization that we can make to share the download task if that download has not completed before the paywall displays.